views/index.ejs

```diff
@@ -1,49 +1,71 @@
-        <div class="container">
-            <div class="grid-container">
-                <% alleHuizen.forEach((huis, index) => { %>
-                    <div class="card-favorites">
-                        <picture>
-                            <source srcset="https://fdnd-agency.directus.app/assets/<%= huis.poster_image %>?format=webp" type="image/webp">
-                            <source srcset="https://fdnd-agency.directus.app/assets/<%= huis.poster_image %>?format=avif" type="image/avif">
-                            <img src="https://fdnd-agency.directus.app/assets/<%= huis.poster_image %>" alt="foto huis" width="400" height="400"  class="house-card">
-                        </picture>  
-
-                        <article>
-                            <ol>
-                                <h2><a class="card-link" href="./huis/<%= huis.id %>"><%= huis.street%> <%= huis.house_nr%></a></h2>
-                                <li>€ <%- huis.price %></li>
-                        
-                                <li><%- huis.postal_code %> <%- huis.city %></li>
-                                <li><%- huis.m2 %> m² • <%- huis.rooms %> Kamers</li>
-                            
-                                <li><%- huis.m2 %></li>
-                                <li><%- huis.agent %></li>
-
-                                <div class="btn__container">
-                                    <a class="btn-popup" href="">Rate dit huis</a>
-                                </div>
-                            </ol>
-                        </article> 
-                    </div>
-                <% }) %>
+  <main>
+    <header>
+      <h1>Favorieten</h1>
+      <hr />
+      <p>In dit overzicht vind je al jouw favoriete huizen terug.</p>
+    </header>
+ 
+    <div class="container">
+      <div class="grid-container">
+        <% alleHuizen.forEach((huis, index) => { %>
+        <div class="card-favorites">
+          <picture>
+            <source
+              srcset="https://fdnd-agency.directus.app/assets/<%= huis.poster_image %>?format=webp"
+              type="image/webp" />
+            <source
+              srcset="https://fdnd-agency.directus.app/assets/<%= huis.poster_image %>?format=avif"
+              type="image/avif" />
+            <img
+              src="https://fdnd-agency.directus.app/assets/<%= huis.poster_image %>"
+              alt="foto huis"
+              width="400"
+              height="400"
+              class="card-image" />
+          </picture>
+ 
+          <article>
+            <ol>
+              <h2><a class="card-link" href="./huis/<%= huis.id %>"><%= huis.street %> <%= huis.house_nr %></a></h2>
+              <li>€ <%- huis.price %></li>
+              <li><%- huis.postal_code %> <%- huis.city %></li>
+              <li><%- huis.m2 %> m² • <%- huis.rooms %> Kamers</li>
+              <li><%- huis.m2 %></li>
+              <li><%- huis.agent %></li>
+ 
+              <div class="btn__container">
```

Comment:
door deze div pop up in de foreach loop te hebben word er 25x een pop up gemaakt